tutorial/source/tracking_1d_em.ipynb

```diff
@@ -296,24 +254,23 @@
     "        # E-step: compute soft assignments\n",
     "        with torch.no_grad():\n",
     "            assign_logits = compute_assign_logits(positions, observations, args)\n",
-    "            exists_logits = torch.empty(args.max_num_objects).fill_(\n",
-    "                math.log(args.expected_num_objects / args.max_num_objects))\n",
+    "            exists_logits=full_exists_logits[:states_loc.shape[0]]\n",
```

Comment:
Hmm I think it would make more sense to fill this with `args.expected_num_objects / states_loc.shape[0]` instead of `args.expected_num_objects / args.max_num_objects`. Does that seem right to you?